ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/components/DeliveryDestinationsDetails.tsx

```diff
@@ -0,0 +1,71 @@
+import {
+    DescriptionList,
+    DescriptionListDescription,
+    DescriptionListGroup,
+    DescriptionListTerm,
+    Flex,
+    FlexItem,
+    Title,
+} from '@patternfly/react-core';
+import React, { ReactElement } from 'react';
+
+import { ReportFormValues } from 'Containers/Vulnerabilities/VulnerablityReporting/forms/useReportFormValues';
+
+export type DeliveryDestinationsDetailsProps = {
+    formValues: ReportFormValues;
+};
+
+function DeliveryDestinationsDetails({
+    formValues,
+}: DeliveryDestinationsDetailsProps): ReactElement {
+    const deliveryDestinations =
+        formValues.deliveryDestinations.length !== 0 ? (
+            formValues.deliveryDestinations.map((deliveryDestination) => (
+                <li key={deliveryDestination.notifier?.id}>{deliveryDestination.notifier?.name}</li>
+            ))
+        ) : (
+            <li>None</li>
+        );
+
+    const mailingLists =
+        formValues.deliveryDestinations.length !== 0 ? (
+            formValues.deliveryDestinations.map((deliveryDestination) => {
+                const emails = deliveryDestination?.mailingLists.join(', ');
+                return <li key={emails}>{emails}</li>;
+            })
+        ) : (
+            <li>None</li>
+        );
```

Comment:
Maybe I'm not understanding how multiple notifiers will work, but this strikes me as potential confusing. The `deliveryDestination` has both a notifier and a mailingList associated with it, but the design doesn't emphasize the connection, and the wider the screen, the harder it would be for a user to scan from the notifier name on the left to its corresponding list of emails on the right.

nit: Would using something more generic than `emails` make it easier to add other types of Notifiers in the future, like Slack, or would that require major changes anyway?